packages/console/package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "@logto/phrases": "^0.1.0",
     "@logto/schemas": "^0.1.0",
+    "@monaco-editor/react": "^4.3.1",
```

Comment:
`monaco-editor` is a peer dependency of this

Reply:
updated